### DISCUSSION TITLE
Passive Vulnerability Management & Asset Inventory

### DISCUSSION THREAD

**Discussion by epineto** [Security-Onion-Solutions/securityonion#6530] [Unsupported Versions] [answered]
Hi folks, I have been working/testing on Security Onion few weeks ago. I am in InfoSec and Cyber fields nearly 12 years and now I am involved with OT/IoT cyber security in ICS industry.

Security Onion can be a powerful SIEM and Security Management System, however I would like to know if there is something on the SO Roadmap related to Passive Vulnerability Assessment/Management on the Security Onion Console (SOC) and some kind of Asset Inventory/Management section where we could organise and create group of assets based on the mapped/identified IPs / devices and then manage them as a baseline aligned with the asset group defined by business, for example.

Currently on the SOC we have "Alerts" and "Hunt" as core functions and It would be awesome if there were "Assets" and "Vulnerabilities" following the idea from my previous comments - everything passively based on the logs collected - however in Assets section we could rename and create groups and in Vulnerabilities section we could "tick" those ones that have been remediated or something like that. And in this Vulnerabilities section, the Security Onion would have a "native" vulnerabilities database and something via API Key to keep it up to date every time that a new zero day / CVE is published.

Apologies for being too long, but does anyone have similar thoughts, suggestion and experiences to share ideas to achieve those suggestion with Security Onion based on my comments?

Thank you and I am looking forward to receiving my hardcopy version of Security Onion released recently and stressing the tool and plugins more and more.

Best regards,

Epi

**Comment by weslambert** [comment, marked as answer]
Wazuh offers some capabilities around determining if there is vulnerable software present on an endpoint (and osquery/Fleet have just rolled out some preliminary features), however, we don't track or manage that information inside of Security Onion.  We don't have anything on the roadmap for encompassing the entirety of asset or vulnerability management, but it may be something to which we consider providing additional capabilities in the future. Another option would be integrating VulnWhisperer with Security Onion and creating your own Kibana dashboards to track that information.

**Reply by epineto** [reply]
Thank you very much!!
